test/pipelinerun_test.go

```diff
@@ -96,10 +96,12 @@ func TestPipelineRun(t *testing.T) {
 			}
 
 			task := tb.Task(getName(taskName, index), namespace, tb.TaskSpec(
+				tb.TaskInputs(tb.InputsParamSpec("path", v1alpha1.ParamTypeString),
+					tb.InputsParamSpec("dest", v1alpha1.ParamTypeString)),
 				// Reference build: https://github.com/knative/build/tree/master/test/docker-basic
 				tb.Step("config-docker", "quay.io/rhpipeline/skopeo:alpine",
 					tb.Command("skopeo"),
-					tb.Args("copy", "docker://gcr.io/build-crd-testing/secret-sauce", "dir:///tmp/"),
+					tb.Args("copy", "${inputs.params.path}", "${inputs.params.dest}"),
```

Comment:
Intentionally using deprecated format here? If so, worth adding a `TODO(#1170)`?

Reply:
good catch - and actually I meant to mix the formatting but apparently forgot to 🤦‍♀ !